packages/components/pie-modal/compsizer.config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "exclude": [
+    "**/*.d.ts"
+  ],
+  "compression": {
+    "gzip": true,
+    "brotli": true
+  },
+  "baselineFile": "component-bundle-sizes.json",
+  "components": {
+    "pie-modal": {
+      "maxSize": "50 KB",
+      "warnOnIncrease": "10%",
+      "distFolderLocation": "./dist"
+    }
+  },
+  "defaults": {
+    "warnOnIncrease": "5%"
+  }
+}
```

Comment:
going to look to making most of these default values in the tool when not provided. it'll make the config object much smaller